Gemfile

```
@@ -11,4 +11,4 @@ platforms :mingw, :x64_mingw, :mswin, :jruby do
   gem "tzinfo-data"
 end
 
-gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
+gem "wdm", ">= 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
```

Thank you for providing this important addition for the Windows platform. Wdm released version 0.2.0 in August this year, so it would be better to change the constraint to `~> 0.2.0`.

```suggestion
gem "wdm", "~> 0.2.0", :platforms => [:mingw, :x64_mingw, :mswin]
```

---

@supreethmv Why did you mark it as resolved?

---

My bad, changed it to `"wdm", "~> 0.2.0"`